app/scripts/controllers/quota.js

```diff
@@ -102,13 +102,13 @@ angular.module('openshiftConsole')
         $scope.project = project;
 
         DataService.list("resourcequotas", context, function(quotas) {
-          $scope.quotas = quotas.by("metadata.name");
+          $scope.quotas = _.sortBy(quotas.by("metadata.name"), 'name');
```

Comment:
on the data service question I tend to agree, but since we are in the middle of moving the services out to the other repo, I'd rather do that separately from this PR